MediaBrowser.Model/Configuration/EncodingOptions.cs

```diff
@@ -237,6 +238,11 @@ public EncodingOptions()
     /// </summary>
     public bool AllowHevcEncoding { get; set; }
 
+    /// <summary>
+    /// Gets or sets a value indicating whether MJPEG encoding is enabled.
+    /// </summary>
+    public bool AllowMjpegEncoding { get; set; }
```

Comment:
Also, I still think this option could be used beyond trickplay--such as using hw encoding or hw tonemapping with extracted images like chapters or the preview image. In this case maybe it would be better to keep this is EncodingOptions but make the variable name clearer?

Reply:
@nyanmisaka thoughts?